### PR TITLE
Improve testing docs

### PR DESCRIPTION
**Pull request by degrammer**

### CODE REVIEW

**Review by mxsdev — APPROVED** [review]
Awesome!

File: tests/runme/RUNNING_CI.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+- uses: actions/setup-go@v3
+  with:
+   go-version: '>=1.17.0'
+   - run: go version          
+   - name: Install runme
+     run: |
+        go install github.com/stateful/runme@latest
+```
+
+You can check the Runme version and is installed adequately via the following action:
```
Comment (mxsdev) [inline tests/runme/RUNNING_CI.md:157]: Should we include something about running with the grpc runner? See my branch for the relevant changes

File: tests/runme/RUNNING_CI.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+``` { mimeType=text/x-yaml}
+- name: 'Initialize Git submodules'
+    run: |
+      git submodule update --init
+                
+- name: 'Run bats tests'
+    env:
+     NO_COLOR: true
+     FROM_CLI: true
```
Comment (mxsdev) [inline tests/runme/RUNNING_CI.md:177]: Maybe explain then need for `NO_COLOR` and `FROM_CLI`?

**Review by mxsdev — COMMENTED** [review]
LGTM, except for comments. We probably should decide if we want the final version to use gRPC runner or not.

File: tests/runme/RUNNING_CI.md (hunk 1 above)
Comment (degrammer) [reply tests/runme/RUNNING_CI.md:158]: Added

**Review by mxsdev — APPROVED** [review]
LGTM

**Pull request merged**